cmd/fits-loader/fits-loader.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"crypto/tls"
+	"flag"
+	"fmt"
+	tk "github.com/GeoNet/fits/internal/credentials/token"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+)
+
+// version 1.x no longer uses the network code in the DB.
+const vers = "1.0"
+
+var (
+	token   string
+	connStr string
+	dataDir string
+	version bool
+)
+
+var conn *grpc.ClientConn
+
+func initConfig() {
+	flag.StringVar(&connStr, "conn-str", "", "connect string for fits-grpc. eg: localhost:8443")
+	flag.StringVar(&dataDir, "data-dir", "", "path to directory of observation and source files.")
+	flag.StringVar(&token, "token", "", "token id to write to fits-grpc.")
+	flag.BoolVar(&version, "version", false, "prints the version and exits.")
+	flag.Parse()
+
+	if version {
+		fmt.Printf("fits-loader version %s\n", vers)
+		os.Exit(1)
+	}
+}
+
+func main() {
+	var err error
+
+	initConfig()
+
+	conn, err = grpc.Dial(connStr,
+		grpc.WithPerRPCCredentials(tk.New(token)),
+		grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{ServerName: "", InsecureSkipVerify: true})))
```

Comment:
the InsecureSkipVerify should only be used when there is a self signed cert on the server.  This needs to be a devmode compile flag or something similar.